File: assets/reduce_cmp.c
int main() {
    printf("Yes:\n");
    printf("\t5 < 6: %d\n", 5 < 6);
    printf("\t5 <= 6: %d\n", 5 <= 6);
    printf("\t5 == 5: %d\n", 5 == 5);
    printf("\t!(6 <= 5): %d\n", !(6 <= 5));
    printf("\t!(6 < 5): %d\n", !(6 < 5));
    printf("\t6 > 5: %d\n", 6 > 5);
    printf("\t6 >= 5: %d\n", 6 >= 5);
    printf("\t6 != 5: %d\n", 6 != 5);
    printf("No:\n");
    printf("\t6 < 5: %d\n", 6 < 5);
    printf("\t6 <= 5: %d\n", 6 <= 5);
    printf("\t5 == 6: %d\n", 5 == 6);
    printf("\t!(5 < 6): %d\n", !(5 < 6));
    printf("\t!(5 <= 6): %d\n", !(5 <= 6));
    printf("\t5 > 6: %d\n", 5 > 6);
    printf("\t5 >= 6: %d\n", 5 >= 6);
    printf("\t5 != 5: %d\n", 5 != 5);
}

File: verify/reduce/reduce_cmp.py
def verify(*args):
    res = ""
    res += "Yes:\n"
    res += "\t5 < 6: {}\n".format(int(5 < 6))
    res += "\t5 <= 6: {}\n".format(int(5 <= 6))
    res += "\t5 == 5: {}\n".format(int(5 == 5))
    res += "\t!(6 <= 5): {}\n".format(int(not (6 <= 5)))
    res += "\t!(6 < 5): {}\n".format(int(not (6 < 5)))
    res += "\t6 > 5: {}\n".format(int(6 > 5))
    res += "\t6 >= 5: {}\n".format(int(6 >= 5))
    res += "\t6 != 5: {}\n".format(int(6 != 5))
    res += "No:\n"
    res += "\t6 < 5: {}\n".format(int(6 < 5))
    res += "\t6 <= 5: {}\n".format(int(6 <= 5))
    res += "\t5 == 6: {}\n".format(int(5 == 6))
    res += "\t!(5 < 6): {}\n".format(int(not (5 < 6)))
    res += "\t!(5 <= 6): {}\n".format(int(not (5 <= 6)))
    res += "\t5 > 6: {}\n".format(int(5 > 6))
    res += "\t5 >= 6: {}\n".format(int(5 >= 6))
    res += "\t5 != 5: {}\n".format(int(5 != 5))
    return (0, res, "")

data = [
    [],
]
